packages/cli/src/models/files/ZosVersion.ts

```diff
@@ -2,12 +2,10 @@ import _ from 'lodash';
 
 export const ZOS_VERSION = '2';
 
-export function checkVersion(version, where) {
-  if (version == ZOS_VERSION) {
-    return true;
-  } else if (_.isUndefined(version)) {
-    throw Error(`zos version identifier not found in ${where}. This means the project was built with an older version of zos (1.x), and needs to be upgraded. Please refer to the documentation at https://docs.zeppelinos.org for more info.`)
-  } else {
-    throw Error(`Unrecognized zos version identifier ${version} found in ${where}. This means the project was built with an unknown version of zos. Please refer to the documentation at https://docs.zeppelinos.org for more info.`)
-  }
-}
\ No newline at end of file
+export function checkVersion(version: string, where: any): boolean {
+  if (version === ZOS_VERSION) return true;
+  else if (_.isUndefined(version))
```

Comment:
Are we using the same linter rules that we are using in `lib`? If we are, please re-run the linter  :slightly_smiling_face:

Reply:
We were using "similar" settings. I've updated them here so that they are as similar as possible to the master branch settings.